lib/browser/browser-legacy.ts

```diff
@@ -14,11 +14,16 @@ import {eventTargetLegacyPatch} from './event-target-legacy';
 import {propertyDescriptorLegacyPatch} from './property-descriptor-legacy';
 import {registerElementPatch} from './register-element';
 
-Zone.__load_patch('registerElement', (global: any, Zone: ZoneType, api: _ZonePrivate) => {
-  registerElementPatch(global);
-});
+(function(_global: any) {
+_global['__zone_symbol__legacyPatch'] = function() {
```

Comment:
Why is it necessary to add `__zone_symbol__legacyPatch` to the global? Who invokes that function?

Reply:
Yeah, here is the flow.
1. loading `browser-legacy.js`, and register this factory function in global.
2. loading `zone-evergreen.js`, and call this `__zone_symbol__legacyPatch` here, https://github.com/angular/zone.js/pull/1201/files#diff-75082c639aefcc47b3b8df57c8870867R23, so the legacy patch can be loaded after the `common logic of zone` is loaded.